### PR TITLE
Search page item type specific error messages

### PR DESCRIPTION
**Pull request by melniiv**
## Description
When no search results found, in all 3 apps should be shown specific to search item type error message (venue, event, course)

### Closes

**[DEV-XXX](https://helsinkisolutionoffice.atlassian.net/browse/DEV-XXX):**

### CODE REVIEW

File: apps/sports-helsinki/src/domain/search/eventSearch/searchResultList/__tests__/ResultsInfo.test.tsx [hunk 1]
```diff
@@ -4,18 +4,18 @@ import { render, screen } from '@/test-utils';
...
+  const { container } = render(
+    <ResultsInfo itemType="Venue" resultsCount={0} />
+  );
 
   expect(container).toMatchSnapshot();
 });
 
 it('renders no events found text', async () => {
-  render(<ResultsInfo resultsCount={0} />);
+  render(<ResultsInfo itemType="Venue" resultsCount={0} />);
```
Comment (nikomakela) [inline apps/sports-helsinki/src/domain/search/eventSearch/searchResultList/__tests__/ResultsInfo.test.tsx:15]: The test text does not match the result and the tested thing. Each of them could be easily tested here with `it.each<['General', 'Course', 'Venue']>`.

**Review by nikomakela — CHANGES_REQUESTED** [review]
Test each type.

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr333.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr333.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr333.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr333.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr333.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**EVENTS GRAPHQL PROXY is deployed to platta: https://events-graphql-proxy-pr333.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**VENUE GRAPHQL PROXY is deployed to platta: https://venue-graphql-proxy-pr333.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr333.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr333.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr333.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr333.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr333.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**EVENTS GRAPHQL PROXY is deployed to platta: https://events-graphql-proxy-pr333.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr333.dev.hel.ninja :laughing::tada::tada::tada:

**Review by nikomakela — APPROVED** [review]

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr333.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr333.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr333.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr333.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EVENTS GRAPHQL PROXY is deployed to platta: https://events-graphql-proxy-pr333.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**VENUE GRAPHQL PROXY is deployed to platta: https://venue-graphql-proxy-pr333.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**